tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/runtime/api/tests/Values.java

```diff
@@ -148,4 +153,43 @@ public static BArray getTypeIds(BObject bObject) {
         }
         return arrayValue;
     }
+
+    public static Object getMapValue1() {
```

Comment:
Shall we name these methods as `getMapValue` and `getMapValueWithInitialValues` and similarly for the records?